docker-compose.yml

```diff
@@ -46,15 +48,17 @@ services:
       <<: *default-logging
 
   slave:
-    image: pavics/jenkins-ssh-slave:190226
+    image: ${JENKINS_SSH_SLAVE_IMAGE}
     hostname: jenkins_slave
     volumes:
       - slave_home:/home/jenkins
       - ./jenkins-slave/entrypoint_slave:/entrypoint:ro
-      - /var/run/docker.sock:/var/run/docker.sock
+      - ${DOCKER_SOCKET_ON_HOST}:/var/run/docker.sock
     environment:
-      JENKINS_SLAVE_SSH_PUBKEY: $JENKINS_SLAVE_SSH_PUBKEY
-      DOCKER_GROUP_ON_HOST: $DOCKER_GROUP_ON_HOST
+      # use both variants such that any agent/slave source code will work
+      JENKINS_SLAVE_SSH_PUBKEY: ${JENKINS_SSH_SLAVE_PUBKEY}  # old name
```

Comment:
Hein?  Why flip the `SLAVE` and `SSH` in the var name?  You wanted to perserve back-compat with older `env.local`, isn't this breaking it?

```diff
- JENKINS_SLAVE_SSH_PUBKEY: $JENKINS_SLAVE_SSH_PUBKEY
+ JENKINS_SLAVE_SSH_PUBKEY: ${JENKINS_SSH_SLAVE_PUBKEY}
``` 

Same the for the new `AGENT` flavor, why flip `AGENT` and `SSH` in the var name?